numpy/core/setup.py

```diff
@@ -883,34 +872,39 @@ def generate_umath_c(ext, build_dir):
             join('src', 'umath', 'scalarmath.c.src'),
             join('src', 'umath', 'ufunc_type_resolution.c'),
             join('src', 'umath', 'override.c'),
-            join('src', 'private', 'mem_overlap.c'),
-            join('src', 'private', 'npy_longdouble.c'),
-            join('src', 'private', 'ufunc_override.c')]
+            # join('src', 'common', 'mem_overlap.c'),
```

Comment:
Why the commented out files? Just remove? Or at least comment on why they are commented out... Is it because they are already listed in `multiarray`?

Reply:
removed